asciidoc-release-notes/adoc/about-documentation.adoc

```diff
@@ -31,4 +36,4 @@ endif::[]
 ifeval::["{lifecycle}" == "unmaintained"]
 * For the most up-to-date version of the documentation for {product}{nbsp}{this-version}, see {doc-url}.
 endif::[]
-* Find a collection of White Papers in the {product} Resource Library at https://www.suse.com/products/server#resources.
\ No newline at end of file
+* Find a collection of White Papers in the {product} Resource Library at https://www.suse.com/products/server#resources.
```

Comment:
Are you sure you want this link in literal form? I would suggest adding an AsciiDoc attribute like `{reslib-url}` to product-generic.adoc for this.

Reply:
That is not a new line in this commit. It's just that the .editorconfig configuration adds a newline at the end of files and there wasn't one before. If you disable whitespace differences it's not highlighted anymore.

Reply:
It's a good point, though. If you don't mind I will change it in this otherwise unrelated PR.

Reply:
Thanks for fixing!

>If you disable whitespace differences it's not highlighted anymore.

I didn't realize that's an option in GH. (I found it, behind the cogwheel.)